app/views/agent/SignUpConfirmation.scala.html

```diff
@@ -73,17 +82,40 @@ <h3 class="govuk-heading-m">@messages("agent.sign-up-confirmation.section1.quart
     </div>
 }
 
-@checkClientDetails = {
-    <div class="govuk-body client-details govuk-!-padding-3 govuk-!-border-0">
-        <h2>
-            @{messages("agent.sign-up-confirmation.check.heading")}
-        </h2>
-        <p>
-            @{Html(messages("agent.sign-up-confirmation.check.description", appConfig.agentServicesAccountHomeUrl))}
-        </p>
+@periodStatement = {
+  <div>
+  @if(taxYearSelectionIsNext) {
+    <h3 class="govuk-heading-m">@messages("agent.sign-up-confirmation.send.end-of-period-statement.next-year.heading")</h3>
+    <p class="govuk-body">@messages("agent.sign-up-confirmation.send.end-of-period-statement.next-year.paragraph", languageUtils.Dates.formatDate(AccountingPeriodUtil.getEndOfPeriodStatementDate(true)))</p>
+  } else {
+    <h3 class="govuk-heading-m">@messages("agent.sign-up-confirmation.send.end-of-period-statement.this-year.heading")</h3>
+    <p class="govuk-body">@messages("agent.sign-up-confirmation.send.end-of-period-statement.this-year.paragraph", languageUtils.Dates.formatDate(AccountingPeriodUtil.getEndOfPeriodStatementDate(false)))</p>
+  }
+  </div>
+}
+
+@finalDeclaration = {
+    <div>
+    @if(taxYearSelectionIsNext) {
+      <h3 class="govuk-heading-m">@messages("agent.sign-up-confirmation.send.final.declaration.next-year.heading")</h3>
+      <p class="govuk-body">@messages("agent.sign-up-confirmation.send.final.declaration.next-year.paragraph", languageUtils.Dates.formatDate(AccountingPeriodUtil.getFinalDeclarationDate(true)))</p>
```

Comment:
For clarity, I'd suggest
```suggestion
      <p class="govuk-body">@messages("agent.sign-up-confirmation.send.final.declaration.next-year.paragraph", languageUtils.Dates.formatDate(AccountingPeriodUtil.getFinalDeclarationDate(taxYearSelectionIsNext)))</p>
```

Reply:
Might even be better to have a little local variable which can be used in both?  Not sure.